tmt/utils.py

```diff
@@ -747,6 +748,18 @@ def environment_to_dict(variables):
     return result
 
 
+@contextlib.contextmanager
+def modify_environ(**new_elements):
```

Comment:
The `new_elements` parameter is not used in `base.py`, right? It's here just for possible future use?

Reply:
Exactly. I don't know when or if ever it will come in handy, it wasn't useful for the parametrization (due to priorities and overriding) but it doesn't get in the way and could simplify the modification of `os.environ` if there is a case where we want to override an existing variable.

Reply:
Understood, thanks for clarification.